packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1149,13 +1151,15 @@ class Auth {
      */
     async getPasswordLessAccessToken(parameters: LoginPasswordlessParams) {
         const pwdlessLoginToken = parameters.pwdlessLoginToken
+        const dntPref = this.getDnt({includeDefaults: true})
         const token = await helpers.getPasswordLessAccessToken(
             this.client,
             {
                 clientSecret: this.clientSecret
             },
             {
-                pwdlessLoginToken
+                pwdlessLoginToken,
+                dnt: String(dntPref)
```

Comment:
Hmm, it's strange that the `dnt` parameter for `helpers.getPasswordLessAccessToken` is of type string, which is different than what's seen with the other helper methods (they use boolean type).

Oh I see now.. behind the scene, the isomorphic sdk converts the dnt value to a string: https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/7e1c96ec53d6808a9ad760f99be6cb26cef09e75/src/static/helpers/slasHelper.ts#L348

Reply:
It's possible for dntPref to be undefined. If so, we should not set the `dnt` parameter as 'undefined' string.
```suggestion
                dnt: dntPref !== undefined ? String(dntPref) : undefined
```